docs/concepts/env_vars.md

```diff
@@ -0,0 +1,115 @@
+---
+sidebar_position: 4
+---
+
+# Variables
+
+Environment variables are key-value pairs defined to match a specific application environment (local, dev, prod, etc.). Often they contain 
+- Flags that control the application behavior
+- Secrets and api keys that can differ by environment
+
+## Setting Variables
+### Local
+:::danger Do not put your `.env` file in GitHub
+Especially if your repo is public. Make sure to add `.env` file to your `.gitignore`. If you accidentally push your `.env` file to GitHub, its contents will be permanently in the repository's history - **even if you delete it afterwards**
+:::
+
+On local, there are many ways to get environment variables into your application. Refer to these popular packages for more info:
+- env-cmd - <a href="https://www.npmjs.com/package/env-cmd" target="_blank">https://www.npmjs.com/package/env-cmd</a> (recommended)
```

Comment:
Does this package require node 18 re: the watch flag, if so we should mention it.

Reply:
👍 added it as a tip